www/annotationdb/annotationdb.js

```diff
@@ -0,0 +1,382 @@
+				   '</a>';
+
+		}
+
+		$('#' + table_id + '>tbody').append(
+			'<tr>' +
+				'<td class="col0">' + col0 + '</td>' +
+				'<td class="col1"><span class="hail-code">' + col1 + '</span></td>' +
+				'<td class="col2"><span class="hail-code">' + col2 + '</span></td>' +
+				'<td class="col3"><span>' + col3 + '</span></td>' +
+			'</tr>'
+		);
+
+		if (has_elements) {
+
+			$('#' + table_id + '>tbody').append(
+				'<tr id="row-' + sub_table_id + '" class="collapse">' +
+					'<td colspan="4" class="sub-table">' +
+						'<table id="' + sub_table_id + '" class="table table-hover sub-table">' + 
+							'<thead>' +
+								'<th>Struct Element</th>' +
+								'<th>Type</th>' +
+							'</thead>' +
+							'<tbody>' +
+							'</tbody>' +
+						'</table>' +								
+					'</td>' +
+				'</tr>'
+			);
+
+			$.each(child_node['elements'], function(_, element){
+				$('#' + sub_table_id + '>tbody').append(
+					'<tr>' +
+						'<td class="sub-col0"><span class="hail-code">' + element['name'] + '</span></td>' +
+						'<td class="sub-col1"><span class="hail-code">' + element['type'] + '</span></td>' +
+					'</tr>'
+				);
+			});
+		}
+
+	});
+	
+}
+
+function build_docs(nodes, parent_id, parent_level) {
+
+	var level = parent_level + 1;
+	var current_parent = parent_id;
+
+	$.each(nodes, function(_, node) {
+
+		var node_id = current_parent + '-' + node['annotation'].split('.').slice(-1)[0];
+
+		$('#' + current_parent).append(
+			'<div class="panel panel-default level' + String(level) + '">' +
+		    	'<div class="panel-heading" id="heading-' + node_id + '">' +
+		          	'<a role="button" data-toggle="collapse" href="#' + node_id + '">' +
+				       	'<span class="text-expand">' + node['text'] + '</span>' +
+				   	'</a>' +
+			   	'</div>' +
+			   	'<div class="panel-collapse collapse" id="' + node_id + '">' +
+			      	'<div class="panel-body">' +
+					'</div>' +
+				'</div>' +
+			'</div>'
+		);
+
+		if (node['free_text']) {
+
+			var paragraphs = node['free_text'].split("\n\n");
+			var insert = '<div class="panel-text">'
+
+			$.each(paragraphs, function(_, paragraph) {
+				insert += '<p>' + paragraph.replace('\n', '<br>') + '</p>'
+			});
+
+			$('#' + node_id + '>.panel-body').append(insert + '</div>');
+
+		}
+
+		if (node['study_title'] && node['study_link']) {
+
+			$('#' + node_id + '>.panel-body').append(
+				'<div class="panel-text">' + 
+				   '<p><a href="' + node['study_link'] + '" target="_blank">' + node['study_title'] + '</a></p>' +
+				'</div>'
+			);
+
+		}
+		
+		if (node['study_data']) {
+
+			$('#' + node_id + '>.panel-body').append(
+				'<div class="panel-text">' +
+				   '<p><a href="' + node['study_data'] + '" target="_blank">Data source</p></a>' +
+				'</div>'
+			);
+
+		}
+
+		if (!node.hasOwnProperty('nodes')) {
+
+			var table_id = node_id + '-table';
+
+			$('#' + node_id + '>.panel-body').append(
+				'<table id="' + table_id + '" class="table table-hover" data-detail-view="true">' +
```

Comment:
The white box was intentional, reason being to allow viewing long annotation names that overflow the annotation table column. Open to suggestions though if you think there's a better way to tackle this issue. I wanted to limit the width of the annotation column because many of the names are not long, and I think that space is better used for the description column.